### PR TITLE
UWP usage detailed doc

### PR DESCRIPTION
**Pull request by fmrico**
Hi @esteve ,

This text describes in detail how to use the generated DLLs in Unity/Hololens.

If you consider any more appropriate organization, please, let me know.

Hope it helps

### CODE REVIEW

**Comment by esteve** [comment]
Thank you so much! I'll give it a look, but unfortunately I'll be busy the next couple of days. If i don't say anything, please ping me again.

File: README.md [hunk 1]
```diff
@@ -145,3 +145,104 @@ ros2 run rcldotnet_examples rcldotnet_listener
...
+right click on Universal Windows project - Add - Existing item...
+```
+and include every DLL file from `{your_ros2_uwp_ws}\install\bin`. Now select all of these files in Solution Explorer and check/set the properties:
+```
+Build action: Content
+Copy to output directory: Copy always
+```
+This allows you to retrieve the files in the same directory as the assembly.
+
+Nest step, in Solution Explorer panel:
```
Comment (esteve) [inline README.md]: Next

File: README.md [hunk 2]
```diff
@@ -145,3 +145,104 @@ ros2 run rcldotnet_examples rcldotnet_listener
...
```
Comment (esteve) [inline README.md]: Next

File: README.md [hunk 3]
```diff
@@ -145,3 +145,104 @@ ros2 run rcldotnet_examples rcldotnet_listener
...
+```
+right click on References - Add reference...
+```
+and include `{your_ros2_uwp_ws}\install\lib\rcldotnet\dotnet\rcldotnet_assemblies.dll`, `{your_ros2_uwp_ws}\install\lib\rcldotnet\dotnet\rcldotnet_common.dll` and `{your_ros2_uwp_ws}\install\lib\std_msgs\dotnet\std_msgs_assemblies.dll`.
+
+Finally compile your project and run it on HoloLens Emulator or HoloLens physical device.
```
Comment (esteve) [inline README.md]: This should mention that for `ros2_uwp` to work on the Hololens, it must be compiled for 32 bits.

Comment (fmrico) [reply README.md]: added

**Comment by esteve** [comment]
Thanks!

**Comment by fmrico** [comment]
Thanks to you for your comments ;)

**Pull request merged**